pkg/assembler/clients/operations/trees.graphql

```diff
@@ -172,6 +179,32 @@ fragment AllSLSATree on HasSLSA {
   }
 }
 
+fragment AllCertifyLegalTree on CertifyLegal {
```

Comment:
[client side path queries](https://github.com/guacsec/guac/blob/main/pkg/assembler/clients/operations/path.graphql) need to be updated to add in `AllCertifyLegalTree`?

Reply:
Done